src/api/useFetch.js

```diff
@@ -60,18 +60,21 @@ const _useFetch = ({
     isLoading: true,
     error: null,
   })
+  const honourCacheKey = React.useRef(true)
```

Comment:
The change to the `_useFetch` function does not correlate to the direct linking to the rescoring UI, does it?

Reply:
component page has to be refreshed